update profiling doc for AITK (#9644)
Microsoft/vscode-docs: docs/intelligentapps/profiling.md

![Screenshot that shows the report data](./images/profiling/by-model-file-result.png)

### Benchmark time for each operation

If OP Profiling is enabled, op level data will be generated to allow you to inspect the model in more detail.

![Screenshot that shows the succeeded notification with OP profiling enabled](./images/profiling/by-model-file-op-succeeded.png)

The report contains detailed latencies for each op.

![Screenshot that shows the report data for each OP](./images/profiling/by-model-file-op-result.png)

> ## Profile an ONNX GenAI Model

Reviewer: ```suggestion
## Profile an ONNX GenAI model
```

In this mode, the profiling tool starts profiling an ONNX GenAI model on a target execution provider (EP) for a specified number of prompts. You can see the resource usage while it's running.

![Screenshot that shows how to start by genai model](./images/profiling/by-genai-model-file-config.png)

> [!NOTE]

> > You need to select the folder of the GenAI Model, i.e. the folder that contains `genai_config.json`.

Reviewer: ```suggestion
> You need to select the folder of the GenAI model, this is the folder that contains `genai_config.json`.
```
